### DISCUSSION TITLE
Missing value error while missing values can be handled by LightGBM

### DISCUSSION THREAD

**Discussion by wregter** [Nixtla/mlforecast#248] [General]
As far as I know LightGBM can handle missing values which could be very useful in some use cases. However, currently I am receiving a missing value error when I have any missing values when fitting the model. Why is this the case? Is there another process that cannot handle missing values during fitting the model? Or is this just meant as a warning? Could including missing values be a potential new feature?

**Comment by jmoralez** [comment]
Hey @wregter. Can you provide more details? LightGBM can handle missing values and will handle them if they appear when predicting (a warning is issued though). An error would only be raised if you have another model that can't handle them, like LinearRegression from scikit-learn for example.

**Reply by wregter** [reply]
Hi @jmoralez, thanks for the quick reply! It was a mistake on my end, sorry for raising this discussion, I will close it now.
